api/filter/http/buffer.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package envoy.api.v2.filter.http;
+
+import "google/protobuf/duration.proto";
+
+// The buffer filter is used to stop filter iteration and wait for a fully
+// buffered complete request. This is useful in different situations
+// including protecting some applications from having to deal with partial
+// requests and high network latency.
+message Buffer {
+  // The maximum request size that the filter will before the connection
+  // manager will stop buffering and return a 413 response.
+  uint32 max_request_bytes = 1;
```

Comment:
Default?

Reply:
I am just going by the documentation. Copy pasted from docs.

Reply:
No defaults. These are required fields

Reply:
So, we have to use wrapped types to ensure that these are always present..